mappings/net/minecraft/client/sound/ReusedAudioStream.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4856 net/minecraft/client/sound/ReusedAudioStream
```

Comment:
What do you mean by `Resused`? Shouldn't it be `Reusable`?

Reply:
@Runemoro because it automatically skips back to the start of the delegate audio stream when the delegate audio stream is empty, hence `reused` not `reusable`, maybe `reusing`?